src/index.html

```diff
@@ -11,8 +11,53 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <div class="product__top">
+        <img
+          class="photo"
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+
+      <div class="product__info">
+        <h1 class="name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the page, so you need to use the another tag here
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina
          <br />
          5K Monoblock (MNED2UA/A)
        </h2>
```